docs/requirements.md

```diff
@@ -2,6 +2,14 @@
 
 This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
 
+## Common requirements
+
+This section describes common requirements for the underlying databases when using ScalarDB.
+
+### Privileges to access the underlying databases
+
+ScalarDB operates the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. Thus, ScalarDB requires a fully privileged account to access the underlying databases.
```

Comment:
```suggestion
ScalarDB operates the underlying databases not only for CRUD operations but also for performing operations like creating or altering schemas, tables, or indexes. Thus, ScalarDB basically requires a fully privileged account to access the underlying databases.
```

I think a bit limited privileges are enough for ScalarDB, although it's not easy to describe the details in this section. Just adding `basically` or something might be an option?

Reply:
Yeah, that's better. Thank you!